L1Trigger/L1CaloTrigger/python/L1CaloJetProducer_cfi.py

```diff
@@ -0,0 +1,164 @@
+import FWCore.ParameterSet.Config as cms
+
+L1CaloJetProducer = cms.EDProducer("L1CaloJetProducer",
+    debug = cms.bool(False),
+    HcalTpEtMin = cms.double(0.5),
+    EcalTpEtMin = cms.double(0.5),
+    HGCalHadTpEtMin = cms.double(0.5),
+    HGCalEmTpEtMin = cms.double(0.5),
+    HFTpEtMin = cms.double(0.5),
+    EtMinForSeedHit = cms.double(2.5), # was 2.5
```

Comment:
The current parameters is the same as "was"? Just to confirm if 2.5 is what you expect.

Reply:
@vshang Can you confirm 2.5 is the good number?

Reply:
The EtMinForSeedHit value should not have changed from 2.5 since I've worked on the code. The comment "was 2.5" was probably left over from the original author of the code.